tests/e2e/support/page/publicLinkPage.ts

```diff
@@ -0,0 +1,37 @@
+import { BrowserContext, Page } from 'playwright'
+import { state } from '../../cucumber/environment/shared'
+import { lastCreatedPublicLink } from './files/publicLink'
+import { expect, Locator } from '@playwright/test'
+
+export class PublicLinkPage {
+  public context: BrowserContext
+  private static page: Page
+  private readonly publicLinkLogInButtonLocator: Locator
```

Comment:
it is not used? Maybe you wanted to add constructor like in PublicLink?
it would be nice do like in PublicLink

Reply:
ok.I will work on that